hls-plugin-api/hls-plugin-api.cabal

```diff
@@ -20,6 +20,11 @@ flag pedantic
   default:     False
   manual:      True
 
+flag use-fingertree
```

Comment:
```suggestion
-- This flag can be used to avoid the dependency on hw-fingertree.
-- We can set this temporarily if we have problems building hw-fingertree
-- for a new version of GHC.
flag use-fingertree
```